src/client/ui/status-bar/progress-bar/determinate-behavior.js

```diff
@@ -0,0 +1,52 @@
+import hammerhead from '../../deps/hammerhead';
+import testCafeCore from '../../deps/testcafe-core';
+
+var shadowUI      = hammerhead.shadowUI;
+var nativeMethods = hammerhead.nativeMethods;
+var styleUtils    = testCafeCore.styleUtils;
+
+
+const DETERMINATE_STYLE_CLASS   = 'determinate';
+const ANIMATION_UPDATE_INTERVAL = 10;
+
+
+export default class DeterminateBehavior {
```

Comment:
`DeterminateIndicator` and `IndeterminateIndicator`. `DeterminateBehavior` is ambigious